src/Transition/CreateNewIDFUsingRulesV9_2_0.f90

```diff
@@ -431,6 +431,14 @@ SUBROUTINE CreateNewIDFUsingRules(EndOfFile,DiffOnly,InLfn,AskForInput,InputFile
               ! If your original object starts with E, insert the rules here
 
               ! If your original object starts with F, insert the rules here
+             CASE('FOUNDATION:KIVA')
+                 CALL GetNewObjectDefInIDD(ObjectName,NwNumArgs,NwAorN,NwReqFld,NwObjMinFlds,NwFldNames,NwFldDefaults,NwFldUnits)
+                 nodiff = .false.
+                 OutArgs(1) = InArgs(1)
+                 OutArgs(2) = Blank
+                 OutArgs(3:CurArgs+1) = InArgs(2:CurArgs)
+                 CurArgs = CurArgs + 1
+                 NoDiff = .false.
```

Comment:
This needs to be described in the src/Transition/InputRules/Rules9-1-0-to-9-2-0.md file as well.